schemas/iso19139/src/main/plugin/iso19139/update-fixed-info.xsl

```diff
@@ -288,10 +290,10 @@
   </xsl:template>
 
   <!-- Add required gml attributes if missing -->
-  <xsl:template match="gml:Polygon[not(@gml:id) and not(@srsName)]|
-                       gml:MultiSurface[not(@gml:id) and not(@srsName)]|
-                       gml:LineString[not(@gml:id) and not(@srsName)]|
-                       gml320:Polygon[not(@gml320:id) and not(@srsName)]">
+  <xsl:template match="gml:Polygon[not(@gml:id) or not(@srsName)]|
+                       gml:MultiSurface[not(@gml:id) or not(@srsName)]|
+                       gml:LineString[not(@gml:id) or not(@srsName)]|
+                       gml320:Polygon[not(@gml320:id) or not(@srsName)]">
```

Comment:
With this change the `gml:id` or `srsName` are going to be replaced if 1 of the values are not filled, should not be checked if are filled to copy them instead of setting the value?